kit/score/score.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package score;
+option go_package = "github.com/autograde/quickfeed/kit/score";
+
+import "patch/go.proto";
+
+// Score give the score for a single test named TestName.
+message Score {
+    uint64 ID = 1;
+    uint64 SubmissionID = 2;
+    string Secret = 3;       // the unique identifier for a scoring session
```

Comment:
Would it possibly be better to add a tag to get this field ignored by the database? Because right now we have a `Secret` column filled with the same value: "hidden" for each and every score, seems unnecessary to put it in the database at all.